.travis.yml

```diff
@@ -2,6 +2,12 @@ dist: xenial
 language: rust
 matrix:
   include:
+    - os: linux
```

Comment:
In this change really necessary? If not, it'll just make more diff with the master branch...

Reply:
Not necessary, I will revert it